src/contrib/mlir/core/pass/ngraph_dialect.cpp

```diff
@@ -0,0 +1,108 @@
+//*****************************************************************************
```

Comment:
The file name is a bit misleading. Maybe `ng_dialect_builder` or something similar ?